era_5g_heartbeat/heartbeat_module.py

```diff
@@ -24,14 +18,36 @@
 # Port of the Heartbeat Module server.
 HEARTBEAT_PORT = int(os.getenv("HEARTBEAT_PORT", 5898))
 
+# Middleware robot ID (robot ID).
+MIDDLEWARE_ROBOT_ID = str(os.getenv("MIDDLEWARE_ROBOT_ID", "00000000-0000-0000-0000-000000000000"))
```

Comment:
Just a note - `str(...)` is unnecessary here. The function already returns a string.